msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -193,6 +195,33 @@ public static SilentTokenCommandParameters createSilentTokenCommandParameters(
         return commandParameters;
     }
 
+    public static DeviceCodeFlowCommandParameters createDeviceCodeFlowCommandParameters(
+            @NonNull final PublicClientApplicationConfiguration configuration,
+            @NonNull final OAuth2TokenCache tokenCache,
+            @NonNull String[] scopes){
+
+        final AbstractAuthenticationScheme authenticationScheme = new BearerAuthenticationSchemeInternal();
```

Comment:
Do we actually know if PoP is supported over DCF? Or is there a plan to support in the future?

Reply:
@iambmelt , @rpdome Do you know if PoP is supported over DCF or is there a plan to support it?

Reply:
I think it should be supported as `AuthenticationScheme` only has to do with `/token` endpoint (unless server has a logic to restrict PoP to specific grant types and dcf is explicitly blocked)

Reply:
I guess I don't have enough context to answer this question. I agree with what you describe (that it *should* work, but I think we won't know for sure until we test it e2e.

We could add a comment (i.e. TODO here) and have that as a separate PR/ work item.

Reply:
This is not presently supported by ESTS. PoP support for DCF is a backlog item, that said, let's not make any implementation or design decisions that preclude us from adding this support in the future

Reply:
@rpdome @shahzaibj ^^